test/fixtures/element.point/rotation.json

```diff
@@ -0,0 +1,244 @@
+{
+    "config": {
+        "type": "bubble",
+        "data": {
+            "datasets": [{
```

Comment:
I recently implemented `*.js` fixture (#5777), it may be better to use a JavaScript config instead with a loop to generate the datasets?

```javascript
var angles = Array.apply(null, Array(17)).map(function(v, i) {
    return -180 + i * 22.5;
});

// ...

module.exports = {
// ...

    datasets: ['triangle', 'start', 'rectRot', /* ... */].map(function(style, y) {
        return {
            pointStyle: style,
            data: Array.apply(null, Array(17)).map(function(x) {
                return {x: x, y: y};
            })
    });

//...
```

Reply:
Thanks for the information. I have replaced `*.json` with `*.js`.